ibis/backends/dask/execution/util.py

```diff
@@ -142,3 +252,16 @@ def compute_sorted_frame(
     result = df.assign(**{sort_col_name: temporary_column})
     result = result.set_index(sort_col_name).reset_index(drop=True)
     return result
+
+
+def assert_identical_grouping_keys(*args):
```

Comment:
is this a testing function?

Reply:
No - as you mentioned later, we perform this check in a few places and it is easier to put this function in a central place with a nice error message.